src/stories/Library/Forms/input/Input.tsx

```diff
@@ -6,13 +7,14 @@ export type InputProps = {
   id: string;
   description?: string;
   validation?: string;
+  classNames?: string;
 };
 
 const Input = (props: InputProps) => {
-  const { label, type, id, description, validation } = props;
+  const { label, type, id, description, validation, classNames } = props;
   const invalid = validation ? "true" : "false";
   return (
-    <div className="dpl-input">
+    <div className={clsx("dpl-input", !!classNames && classNames)}>
```

Comment:
```suggestion
    <div className={clsx("dpl-input", classNames)}>
```
I think `clsx()` understands falsy values?